configs/twins/twins_svt-l_fpn_fpnhead_8x4_512x512_80k_ade20k.py

```diff
@@ -0,0 +1,13 @@
+_base_ = ['./twins_svt-s_fpn_fpnhead_8x4_512x512_80k_ade20k.py']
+
+model = dict(
+    pretrained='pretrained/alt_gvt_large.pth',
```

Comment:
This line is redundant, would raise error for `got an unexpected keyword argument pretrained `.